setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='python-saml',
-    version='2.1.6',
+    version='2.2.0',
```

Comment:
Can you keep the 2.1.6 version.

We only modify that value when a release is made, and we just made 1 release.
We will release a new version once we have some new features.